src/index.html

```diff
@@ -3,10 +3,321 @@
   <head>
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap" rel="stylesheet">
     <title>Catalog</title>
     <link rel="stylesheet" href="./styles/main.scss">
   </head>
   <body>
-    <h1>Catalog</h1>
+    <header class="header">
+      <a href="index.html" class="logo">
+        <img
+          src="./images/logo.png"
+          alt="MoyoLogo"
+          class="logo__img"
+        >
+      </a>
+
+      <nav class="nav">
+        <ul class="nav__list">
+          <li class="nav__item">
+            <a href="index.html" class="nav__link is-active">APPLE</a>
+          </li>
+
+          <li class="nav__item">
+            <a href="index.html" class="nav__link">SAMSUNG</a>
+          </li>
+
+          <li class="nav__item">
+            <a href="index.html" class="nav__link">SMARTPHONES</a>
+          </li>
+
+          <li class="nav__item">
+            <a
+              href="index.html"
+              class="nav__link"
+              data-qa="nav-hover"
+            >
+              LAPTOPS & COMPUTERS
+            </a>
+          </li>
+
+          <li class="nav__item">
+            <a href="index.html" class="nav__link">GADGETS</a>
+          </li>
+
+          <li class="nav__item">
+            <a href="index.html" class="nav__link">TABLETS</a>
+          </li>
+
+          <li class="nav__item">
+            <a href="index.html" class="nav__link">PHOTO</a>
+          </li>
+
+          <li class="nav__item">
+            <a href="index.html" class="nav__link">VIDEO</a>
+          </li>
+        </ul>
+      </nav>
+    </header>
+
+    <div class="container">
+      <div class="card" data-qa="card">
+        <a href="#" class="card__link">
+          <img
+            src="./images/imac.jpeg"
+            alt="iMac"
+            class="card__img"
+          >
+        </a>
+
+        <h2 class="card__title">APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)</h2>
+        <p class="card__code">Product code: 195434</p>
+
+        <div class="card__rating">
+          <div class="stars stars--4">
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+          </div>
+
+          <p class="card__rating card__rating--text">Reviews: 5</p>
```

Comment:
```suggestion
          <p class="card__rating-text">Reviews: 5</p>
```

Reply:
it's not card__rating element
and card__rating-text is not modifier(shouldn't be written with '--'
it is element `card__rating-text`